app/lib/actions/pulp3/content_migration.rb

```diff
@@ -9,7 +9,8 @@ def plan(smart_proxy = SmartProxy.pulp_master)
       end
 
       def invoke_external_task
-        migration_service = ::Katello::Pulp3::Migration.new(smart_proxy)
+        migration_service = ::Katello::Pulp3::Migration.new(smart_proxy,
+                                                            smart_proxy.supported_pulp_types[:pulp3][:overriden_to_pulp2])
```

Comment:
ohhh i see,   I think instead of passing in only certain types (meaning if nothing was to be migrated, it would pass in []), we probably instead want to check and actually throw an error.  thoughts?

Reply:
@jlsherrill are you thinking I should add a check for `supported_pulp_types[:pulp3][:overriden_to_pulp2]` being empty and error out if so? But otherwise keep the `migration_service` line the same?

Reply:
compare that to what is being requested to be migrated, but yes, let the migration service dictate what is being attempted to be migrated